forest_lib/forest/data.py

```diff
@@ -275,6 +247,15 @@ def get_available_datasets(s3_base,
     return fcast_time, datasets
 
 
+def get_available_times(datasets, var1):
```

Comment:
I think this has just been moved and should probably dealt with in different PR but not keen on this uber dictionary with instances in it, and you need tor remember all the keys...